print-battleground-state-changes

```diff
@@ -261,17 +261,21 @@ def json_to_summary(
     candidate2 = row.candidates[1] # Trailing candidate
     candidate1_name = candidate1['last_name']
     candidate2_name = candidate2['last_name']
-    candidate1_votes = candidate1['votes']
-    candidate2_votes = candidate2['votes']
-    vote_diff = candidate1_votes - candidate2_votes
+    votes_c = {}
+    votes_c[candidate1_name] = candidate1['votes']
+    votes_c[candidate2_name] = candidate2['votes']
+    if votes_c[candidate1_name] > votes_c[candidate1_name]:
```

Comment:
Should the RHS of this if statement be candidate2_name?

Reply:
votes_c now contains the number of votes...

Reply:
Sorry, I will reword this question;

Should the RHS side of this if statement be "votes_c[candidate2_name]" ?

At the moment the number of votes for candidate1 is being compared to itself.

Reply:
Ah, OK, I see what you mean, it's the last line. Anyway, I'm closing this. I've reviewed other PR and looks much more promising.